.circleci/config.yml

```diff
@@ -3,9 +3,10 @@ version: 2
 jobs:
   deploy:
     docker:
-      - image: google/cloud-sdk:251.0.0
+      - image: google/cloud-sdk:latest
     steps:
       - checkout
+      - run: apt-get install -y build-essential wget
```

Comment:
We probably don't need `wget` anymore. Is there a reason to bring it back?

Reply:
No, I was just reverting back to Greyson's working commit, although `build-essential` is essential.